src/devices/tuya.ts

```diff
@@ -697,6 +697,46 @@ const definitions: Definition[] = [
             return exps;
         },
     },
+    {
+        fingerprint: tuya.fingerprint('TS0601', ['_TZE200_dhke3p9w']),
+        model: 'F00YK04-18-1',
+        vendor: 'FORIA',
+        description: '18 scenes remote',
+        fromZigbee: [tuya.fz.datapoints],
+        toZigbee: [tuya.tz.datapoints],
+        configure: tuya.configureMagicPacket,
+        exposes: [e.battery(), e.battery_voltage(), e.action(['scene_1', 'scene_2', 'scene_3', 'scene_4', 'scene_5', 'scene_6',
+            'scene_7', 'scene_8', 'scene_9', 'scene_10', 'scene_11', 'scene_12', 'scene_13', 'scene_14', 'scene_15', 'scene_16',
+            'scene_17', 'scene_18']),
+        ],
+        endpoint: (device) => {
```

Comment:
I don't think these are needed, does it work without the whole `endpoint`?

Reply:
Without endpoint, it does not work, I checked it out

Reply:
yes, its ok